src/Compilers/VisualBasic/Test/Syntax/packages.config

```diff
@@ -1,8 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.CodeAnalysis.Test.Resources.Proprietary" version="1.0.0-rc3-20150528-03" targetFramework="net45" />
-  <package id="xunit" version="2.0.0-alpha-build2576" targetFramework="net45" />
-  <package id="xunit.abstractions" version="2.0.0-alpha-build2576" targetFramework="net45" />
-  <package id="xunit.assert" version="2.0.0-alpha-build2576" targetFramework="net45" />
-  <package id="xunit.core" version="2.0.0-alpha-build2576" targetFramework="net45" />
+  <package id="xunit" version="2.1.0-beta2-build2981" targetFramework="net45" />
+  <package id="xunit.abstractions" version="2.0.0" targetFramework="net45" />
```

Comment:
xunit.assert and xunit.core should be removable (since they are explicit dependencies of xunit). However, xunit.abstractions is not and should probably be kept (unless we determine the project truly doesn't need it).